### DISCUSSION TITLE
Solved: nzyme with ALFA AWUS036ACS (RTL8811AU)

### DISCUSSION THREAD

**Discussion by crs369** [nzymedefense/nzyme#533] [General]
** NOTE BY nzyme staff: The "After the installation from nzyme do these steps" will no longer be required in nzyme >2.0

Hello Community,

i want to share my success with the ALFA AWUS036ACS (RTL8811AU).

Simply all you need is the great [installation guide](https://www.nzyme.org/docs/installation-configuration/install-raspberry-pi-os/) from lennart and the [WLAN Driver](https://github.com/aircrack-ng/rtl8812au)

Here is a little guide for the Raspberry Pi 4:

```
sudo apt-get install dkms

sudo apt-get install git
git clone -b v5.6.4.2 https://github.com/aircrack-ng/rtl8812au.git
cd rtl*
sudo apt-get install raspberrypi-kernel-headers

sed -i 's/CONFIG_PLATFORM_I386_PC = y/CONFIG_PLATFORM_I386_PC = n/g' Makefile
sed -i 's/CONFIG_PLATFORM_ARM64_RPI = n/CONFIG_PLATFORM_ARM64_RPI = y/g' Makefile
export ARCH=arm
sed -i 's/^MAKE="/MAKE="ARCH=arm\ /' dkms.conf

sudo make dkms_install
```

After the installation from nzyme do these steps:

Change the nzyme.conf to :
`skip_enable_monitor: true`

Because of that, you must enable the monitor mode manually. I have insert the following line in /etc/rc.local:

iw dev <YOUR_WIFI_INTERFACE_NAME> set type monitor

Have fun!

**Comment by lennartkoopmann** [comment]
That's awesome, thank you! I'll add this to the documentation, under a "community guides" section or something.

**Reply by Guenni-at-home** [reply]
In addition it is quite important to set wifi-country afterwards. Otherwise monitor connection will drop within short time and always needs to be restarted again

**Comment by preimann01** [comment]
CSL USB 2.0 WLAN-Adapter 300649 has the monitor mode also. To do it manually on Raspbian GNU/Linux 11 (bullseye) use:

$ sudo ip link set dev xxx down
$ sudo iw dev xxx set type monitor
$ sudo ip link set dev xxx up

Check with

$ iwconfig
xxx IEEE 802.11  Mode:Monitor  Frequency:2.442 GHz

**Comment by lennartkoopmann** [comment]
Thank you! I added a note about v2.x compatibility to the top of the post.
